### PR TITLE
Initial code review 1

### PR DESCRIPTION
**Pull request by BelindaWilliams**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	  <div class="personal">
+	    <p>Full Stack Software Developer and experienced Arts professional</p>
+	    <p>I turn <span>concepts</span> into <span>reality</span></p>
+      </div>
+  </section>
+	
+  <section id="work" class="work">
+    <h2>SELECTED WORK</h2>
+	<div class="project-container">
+	  <div class="project1">
```
Comment (henridvf) [inline index.html:52]: Each project has its own style (project1, 2, 3 in CSS). Do you intend to differentiate between how you style each project?

File: CSS/Portfolio_mystyles.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+.project-container {
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.project1, .project2, .project3 {
```
Comment (henridvf) [inline CSS/Portfolio_mystyles.css]: See comment below in main html file about using different styling for each project.

File: CSS/mystyles.css [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+  background-color: #F5EFE7;
+}
+
+.dropdown {
+  position: fixed;
+  display: inline-block;
+  z-index: 1;
+  padding: 1rem;
+}
+.dropbtn {
```
Comment (tomgarrett95) [inline CSS/mystyles.css:18]: Blank line between rules.

**Review by tomgarrett95 — CHANGES_REQUESTED** [review]
Great stuff - I could only spot one tiny error in your mystyles.css file!

**Review by tomgarrett95 — APPROVED** [review]

**Pull request merged**